src/router/index.js

```diff
@@ -112,4 +111,25 @@ router.beforeEach((to, from, next) => {
 Router auth logic end
 */
 
+// set organization in vuex state if the route org parameter is in vuex user organizations array
+router.beforeEach((to, from, next) => {
+  if (
+    typeof to.params.org !== "undefined" &&
+    store.getters["auth/isAuthenticated"]
+  ) {
+    let orgShortcodes = store.state.auth.user.organizations.map(
+      (organization) => organization.shortcode
+    );
```

Comment:
If a user is added to an organization after they have already logged in, the `user.organizations` will still reflect the value from before the user was added to the org (right after they first logged in) right?

Reply:
Yeah, for that, we probably need to refresh the user data or do a reload after we create the UI for assigning users to an organization.

Reply:
Yeah, they'll have to re-login. Maybe we can do another API call after the organization assignment (from UI when we build it).

Reply:
tracking it in #170